### PR TITLE
Add containerSecurityContext to helm charts

### PR DESCRIPTION
**Pull request by funkypenguin**
Hi guys,

For your consideration, a PR to introduce containerSecurityContexts to the operator/console deployments.

Why?

Some security mitigations can only be applied within the context of the container spec (*such as dropping capabilities, or preventing privileged mode*), and without the ability to configure a container securityContext, validation / security assessment tools such as Kyverno will raise exceptions for operator/console.

Example below:

Policy Reporter UI - Policy Reporter UI 2022-09-05 12-17-25

### CODE REVIEW

File: helm/operator/Chart.yaml
Comment (harshavardhana) [inline helm/operator/Chart.yaml:4]: do not make these changes @funkypenguin changing versions please leave this to maintainers..

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: helm/operator/Chart.yaml
Comment (funkypenguin) [reply helm/operator/Chart.yaml:4]: Got it. I'll revert that.

**Comment by dilverse** [comment]
Hi @funkypenguin thanks for sending this PR! Based on our testing we identified this change breaks Kubernetes 1.21 and we have support up to Kubernetes 1.20. Many of the customers still use Kubernetes 1.21. Once Kubernetes 1.26/1.27 is out we will merge it

File: helm/operator/Chart.yaml
Comment (harshavardhana) [inline helm/operator/Chart.yaml:4]: you shouldn't be changing these @funkypenguin

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by dvaldivia** [comment]
I think this PR failed due to `fsGroup: 1000`  not being a valid field for container SecurityContext, can you update the PR removing that from the values? perhaps that will make this work

**Comment by dilverse** [comment]
Fixed in #1372. Closing this PR

**Pull request closed without merging**
